apph-front/src/index.tsx

```diff
@@ -1,16 +1,38 @@
 import React from 'react';
 import './index.css';
 import reportWebVitals from './reportWebVitals';
-import { SignIn } from './static/components/SignIn';
 import { createRoot } from 'react-dom/client';
+import App from './App';
+import {
+  Route,
+  Routes,
+  Navigate,
+  BrowserRouter as Router
+} from 'react-router-dom';
+import { SignIn } from './static/components/SignIn';
+import { SignUp } from './static/components/SignUp';
 
 const container = document.getElementById('root');
 // eslint-disable-next-line @typescript-eslint/no-non-null-assertion
 const root = createRoot(container!);
+
+function RouterSwitch() {
+  return (
+    <div>
+      <Routes>
+        <Route path="/" element={<App />} />
+        <Route path="/signIn" element={<SignIn />} />
+        <Route path="/signUp" element={<SignUp />} />
+        <Route path="*" element={<Navigate to="/" replace />} />
```

Comment:
Tu devrais rajouter ici des `exact` afin de spécifier qu'on veut ce chemin exactement, et modifier la dernière ligne pour laisser `/`... (à vérifier pour cette dernière modif, mais normalement ça fonctionne comme ça)
```suggestion
        <Route exact path="/" element={<App />} />
        <Route exact path="/signIn" element={<SignIn />} />
        <Route exact path="/signUp" element={<SignUp />} />
        <Route path="/" element={<Navigate to="/" replace />} />
```